src/components/Card/Card.tsx

```diff
@@ -34,10 +34,14 @@ const Card = ({plant}: CardProps) => {
         })
     })
 
+    const draggedCardStyle = isDragging && {opacity: '.4'}
+
     return (
-        <div className='card' ref={dragRef}>
-            <img src={plant.image} alt={`${plant.name}`} />
-            <p className='plant-name'>{plant.name.toUpperCase()}</p>
+        <div className='card' style={{...draggedCardStyle}}>
```

Comment:
If you aren't merging multiple objects, then here you wouldn't need to splat/spread this object out. This one can just be:
```
style={draggedCardStyle}
```

Reply:
It was complaining about that to me, so there's probably another step I'm missing.  When hovering over `style`:
`Type 'false | { opacity: string; }' is not assignable to type 'CSSProperties | undefined'.`

Also I originally splatted it thinking I might add more styles, but I probably won't haha.  Just gonna put it all in-line

Reply:
Oh I see, that would mean that `const draggedCardStyle = isDragging && {opacity: '.4'}` should be:
```
const draggedCardStyle = isDragging ? {opacity: '.4'} : {};
```

Reply:
ahhhh yah that works, that would have taken me forever to debug hahaha